src/orch/opt.h

```diff
@@ -1,6 +1,6 @@
 #ifndef _BLUE_CHIHUAHUA_ORCH_OPTIONS
 #define _BLUE_CHIHUAHUA_ORCH_OPTIONS
 
-void get_opts(int argc, char *argv[], int *ctrlPort);
+int get_opts(int argc, char *argv[], int *ctrlPort, char *configPath);
```

Comment:
Pass in a double pointer to update the pointer in `get_opts`. This is based on the other suggestion. 
```suggestion
int get_opts(int argc, char *argv[], int *ctrlPort, char **configPath);
```